ui/src/app/base/components/NodeSummaryNetworkCard/NodeSummaryNetworkCard.test.tsx

```diff
@@ -0,0 +1,152 @@
+import { mount } from "enzyme";
```

Comment:
This looks like a large diff but it's mainly moving over the file from the machine summary to base components